From: James McCoy <jamessan@debian.org>
Date: Tue, 5 Nov 2024 08:51:15 -0500
Subject: Relax ctor dependency to >= 0.1, < 0.3

Forwarded: not-needed
Signed-off-by: James McCoy <jamessan@debian.org>
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 0ecb634..91dc061 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -49,7 +49,7 @@ clap = { version = "4.4.18", features = [
   "help",
   "unstable-styles",
 ] }
-ctor = "0.2.8"
+ctor = ">= 0.1, < 0.3"
 ctrlc = { version = "3.4.2", features = ["termination"] }
 difference = "2.0.0"
 dirs = "5.0.1"
